integration-tests/http/src/main/java/org/apache/camel/quarkus/component/http/it/GreetingServerEndpoint.java

```diff
@@ -16,14 +16,34 @@
  */
 package org.apache.camel.quarkus.component.http.it;
 
+import javax.websocket.OnClose;
 import javax.websocket.OnMessage;
+import javax.websocket.OnOpen;
+import javax.websocket.Session;
 import javax.websocket.server.ServerEndpoint;
 
+import org.jboss.logging.Logger;
+
 @ServerEndpoint("/ahc-ws/greeting")
 public class GreetingServerEndpoint {
 
+    public static boolean connected = false;
```

Comment:
Shouldn't this be volatile? HttpResource.java is going to read it from a different thread and it would be better it if gets a non-cached value, wouldn't it?

Reply:
Yeah, you're right. Latest commit fixes that.